Fenwick_Tree/Fenwick_Tree.dart

```diff
@@ -0,0 +1,164 @@
+/**
+ *    Fenwick Tree
+ * -------------------------
+ * Binary Indexed tree or Fenwick tree is used to solve
+ * range queries optimally. It has O(n) space complexity,
+ * O(logn) time complexity to update, O(nlog(n)) time complexity to build
+ * and O(logn) time complexity to find sum in a given range.
+ * 
+ * References - Geeksforgeeks
+ * 
+ */
+
+
+// Importing required libraries
+import 'dart:io';
+
```

Comment:
remove this line

Reply:
I removed that, committed and squashed, Please let me know about changes if any further and merge.